src/go-contactless.html

```diff
@@ -130,20 +118,60 @@ <h3>Transport for London</h3>
       </picture>
       <h3>MTA New York City Transit</h3>
       <p>
-        On the first day that MTA launched contactless fares—at just 16 subway stations—nearly 6,000 riders tapped to
-        pay: 80% with a smart device and 20% with a bank card.
+        On the first day that MTA launched contactless fares—at just 16 subway stations—nearly 6,000 riders tapped to pay: 80%
+        with a smart device and 20% with a bank card.
       </p>
     </div>
   </div>
 </section>
 
-<hr>
-
-<section class="go-contactless-section pb-0 text-center" id="call-to-action">
+<section class="go-contactless-section go-contactless-timeline-container bg-dark-tan" id="call-to-action">
   <header>
-    <h2 class="mt-0">Ready to tap into the future of fare payment?</h2>
+    <h2 class="mt-0 mb-4 pb-3 text-center">Ready to tap into the future of fare payment?</h2>
   </header>
-  <section>
-    <a class="btn btn-outline-dark" href="/contracts">Get Started</a>
-  </section>
+  <div class="row mt-4 pt-3 d-flex flex-column-reverse flex-md-row">
```

Comment:
> d-flex flex-column-reverse flex-md-row

Your commit message from cb14278d870b98515b6f437f885b71c6d27beaba says "image on top on mobile", but in Figma, I see the image should be on the bottom? Am I looking at the right comp?

![image](https://user-images.githubusercontent.com/25497886/202315206-327ed6ad-6040-4259-a5a5-40f0226bd100.png)

Reply:
Yea it's the right comp. @segacy1 approved the stacking order in this comment https://github.com/cal-itp/mobility-marketplace/pull/386#issuecomment-1301398426

Reply:
The image on top makes sense to me. I can't find any comment talking about putting the button in the middle, but I'm ok with leaving it, and maybe y'all talked about it somewhere else?

Reply:
Hey! I think I may have missed this. Right now, in netlify I see the stacking as image>button>steps, but could we actually make it image>steps>button? Sorry about that!

Reply:
@segacy1 yes I will!

Reply:
Fixed

Reply:
I see the preview site now with mobile ordering as steps>button>image, which matches the ordering in the design currently in Figma

| Current preview | Current Figma |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/25497886/205144177-0adf1bef-da84-467c-8d10-7d6ef8c61e8e.png) | ![image](https://user-images.githubusercontent.com/25497886/205144260-8e3b5e04-dc5e-4c7b-9736-c0e881852f17.png) |